backend/controllers/user.js

```diff
@@ -0,0 +1,72 @@
+const User = require('../models/User');
```

Comment:
Model `User` ist richtig benannt. Hier im Controller verarbeitest du aber mehrere `User`, also sollte der Controller auch `Users` heißen.

Reply:
Habe es in users Umbenannt :+1: